test/js_test.go

```diff
@@ -793,13 +796,13 @@ func TestJetStreamImport(t *testing.T) {
 			JS: {
 				jetstream: enabled
 				users: [ {user: dlc, password: foo} ]
-				exports [ { service: "$JS.API.>" },  { service: "foo" }]
+				exports [ { service: "$JS.API.>" },  { service: "test" }]
```

Comment:
Temporarily changed since this test failing sometimes locally with a `stream create failed: subjects overlap with an existing stream` error.

Reply:
We need to make sure if we are running server with JS that we always blow away and persisted store directories.

Reply:
OK now should be fixed.